larq_compute_engine/tflite/kernels/quantization.cc

```diff
@@ -72,6 +73,8 @@ TfLiteStatus DequantizePrepare(TfLiteContext* context, TfLiteNode* node) {
 }
 
 TfLiteStatus QuantizeEval(TfLiteContext* context, TfLiteNode* node) {
+  ruy::profiler::ScopeLabel label("Binary quantize");
```

Comment:
I originally named the scope after the op name `LceQuantize`, but that's inconsistent with the scope for our bconv op, which has op name `LceBconv2d`, but scope name `BConv2D`. I'm open to alternative suggestions for names.